.vscode/settings.json

```diff
@@ -376,7 +376,14 @@
     "astro"
   ],
   "[astro]": {
-    "editor.wordWrapColumn": 120
+    "editor.wordWrapColumn": 120,
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
```suggestion
    "editor.defaultFormatter": "astro-build.astro-vscode"
```

Reply:
Will this create a fight between the two?
![image](https://github.com/agrc/gis.utah.gov/assets/1326248/93d8a093-ff90-4b45-bd94-b93a4ce62e2c)

Reply:
They use each other so I don't think so.